MYSQL.txt

```diff
@@ -0,0 +1,2075 @@
+CNUM int PRIMARY KEY,
+CNAME varchar(40),
+CITY varchar(40),
+RATINGS int ,
+SNUM int,
+FOREIGN KEY(SNUM) REFERENCES Salespeople(SNUM) 
+);
+
+-- Insert the data in Customer Table
+
+INSERT INTO Customers VALUES
+(2001,"Hoffman","London",100,1001),
+(2002,"Giovanni","Rome",200,1003),
+(2003,"Liu","SanJose","200",1002),
+(2004,"Grass","Berlin",300,1002),
+(2006,"Clemens","London",100,1001),
+(2008,"Cisneros","SanJose",300,1007),
+(2007,"pereira","Rome",100,1004);
+
+Customers Table
++------+----------+---------+---------+------+
+| CNUM | CNAME    | CITY    | RATINGS | SNUM |
++------+----------+---------+---------+------+
+| 2001 | Hoffman  | London  |     100 | 1001 |
+| 2002 | Giovanni | Rome    |     200 | 1003 |
+| 2003 | Liu      | SanJose |     200 | 1002 |
+| 2004 | Grass    | Berlin  |     300 | 1002 |
+| 2006 | Clemens  | London  |     100 | 1001 |
+| 2008 | Cisneros | SanJose |     300 | 1007 |
+| 2007 | pereira  | Rome    |     100 | 1004 |
++------+----------+---------+---------+------+
+
+
+-- Create a Orders Table
+
+CREATE TABLE Orders(
+ONUM int PRIMARY KEY,
+Amt float ,
+ODATE Date ,
+CNUM int,
+SNUM int,
+FOREIGN KEY(SNUM) REFERENCES salespeople(SNUM)
+);
+
+-- Insert the data into Orders Table
+
+INSERT INTO Orders VALUES
+(3001,18.69,"1990-10-03",2008,1007),
+(3003,767.19,"1990-10-03",2001,1001),
+(3002,1900.10,"1990-10-03",2007,1004),
+(3005,5160.45,"1990-10-03",2003,1002),
+(3006,1098.16,"1990-10-03",2008,1007),
+(3009,1713.23,"1990-10-04",2002,1003),
+(3007,75.75,"1990-10-04",2004,1002),
+(3008,4723.00,"1990-10-05",2006,1001),
+(3010,1309.95,"1990-10-06",2004,1002),
+(3011,9891.88,"1990-10-06",2006,1001);
+
+
+Orders Table 
++------+---------+------------+------+------+
+| ONUM | Amt     | ODATE      | CNUM | SNUM |
++------+---------+------------+------+------+
+| 3001 |   18.69 | 1990-10-03 | 2008 | 1007 |
+| 3002 |  1900.1 | 1990-10-03 | 2007 | 1004 |
+| 3003 |  767.19 | 1990-10-03 | 2001 | 1001 |
+| 3005 | 5160.45 | 1990-10-03 | 2003 | 1002 |
+| 3006 | 1098.16 | 1990-10-03 | 2008 | 1007 |
+| 3007 |   75.75 | 1990-10-04 | 2004 | 1002 |
+| 3008 |    4723 | 1990-10-05 | 2006 | 1001 |
+| 3009 | 1713.23 | 1990-10-04 | 2002 | 1003 |
+| 3010 | 1309.95 | 1990-10-06 | 2004 | 1002 |
+| 3011 | 9891.88 | 1990-10-06 | 2006 | 1001 |
++------+---------+------------+------+------+
+
+
+
+-- Queries
+
+1) List all the columns of the Salespeople table. 
+
+ mysql> SHOW COLUMNS FROM Salespeople;
++-------+-------------+------+-----+---------+-------+
+| Field | Type        | Null | Key | Default | Extra |
++-------+-------------+------+-----+---------+-------+
+| SNUM  | int         | NO   | PRI | NULL    |       |
+| SNAME | varchar(40) | YES  |     | NULL    |       |
+| CITY  | varchar(40) | YES  |     | NULL    |       |
+| COMM  | int         | YES  |     | NULL    |       |
++-------+-------------+------+-----+---------+-------+
+
+
+2) List all customers with a rating of 100. 
+
+
+mysql> SELECT CNUM,CNAME,CITY FROM Customers 
```

Comment:
Follow proper formatting conventions in writing queries. All queries must be formatted according to SQL conventions. Do this change in the entire module.

For example, query should be formatted in this manner - 

SELECT 
    s.sname, s.snum
FROM
    salespeople AS s
        INNER JOIN
    customers AS c ON s.snum = c.snum
GROUP BY s.snum
HAVING COUNT(c.cnum) > 1;